pepys_admin/admin_cli.py

```diff
@@ -84,16 +84,13 @@ def do_status(self):
 
         with self.data_store.session_scope(), handle_status_errors():
             measurement_summary = self.data_store.get_status(report_measurement=True)
-            report = measurement_summary.report()
-            print(f"## Measurements\n{report}\n")
+            measurement_summary.report("Measurements")

```

Comment:
@BarisSari  - so `report()` should not return a parameter now, I guess.

Reply:
We are styling the table and printing it inside of `report()`. But, it still returns not customised text. That's why it was printed twice: one in` report()` (table headers in bold), another with normal` print()` line 88.

Reply:
Hello @BarisSari  - yes, I understand. So, now we should modify `report()` so that it doesn't return a value (including the doc-string).